spec/v1/annotations/object-model.yaml

```diff
@@ -31,6 +31,18 @@ definitions:
         title: Element Reference
         ref: "#/definitions/ElementReference"
 
+  "@ObjectModel.custom":
+    type: boolean
+    description: |-
+      Added through custom development, diverges from the "core" model.
```

Comment:
In my view it is important to make it clear that `@ObjectModel.custom: false` is not the same as annotation `@ObjectModel.custom` not present.

Should we go forward with my suggested change?

Reply:
I've merged it :)